pokete.py

```diff
@@ -1436,23 +1095,23 @@ def map_additions():
 ##############  ##########################
 ##############  ##########################
 ##############  ##########################""", ignore="#",
-                         ob_class=HightGrass,
+                         ob_class=HighGrass,
                          ob_args=_map.poke_args,
                          state="float")
     # adding
     _map.inner.add(_map, 0, 0)
 
     # playmap_3
-    _map = ob_maps["playmap_3"]
-    _map.dor = DorToCenter()
-    _map.shopdor = DorToShop()
+    _map = obmp.ob_maps["playmap_3"]
+    _map.dor = DoorToCenter()
+    _map.shopdor = DoorToShop()
     # adding
     _map.dor.add(_map, 25, 6)
     _map.shopdor.add(_map, 61, 6)
 
     # playmap_4
-    _map = ob_maps["playmap_4"]
-    _map.dor_playmap_5 = ChanceDor("~", state="float",
+    _map = obmp.ob_maps["playmap_4"]
+    _map.dor_playmap_5 = ChanceDoor("~", state="float",
                                    arg_proto={"chance": 6,
                                               "map": "playmap_5",
                                               "x": 17, "y": 16})
```

Comment:
Due to renaming `ChangeDor` to `ChangeDoor`, these lines require one further space of indentation:

```suggestion
                                    arg_proto={"chance": 6,
                                               "map": "playmap_5",
                                               "x": 17, "y": 16})
```